qemu/tests/cfg/numa_node_affinity.cfg

```diff
@@ -0,0 +1,9 @@
+- numa_node_affinity:
+    no RHEL.6 RHEL.7 RHEL.8
+    no Host_RHEL.m6 Host_RHEL.m7 Host_RHEL.m8
+    required_qemu = [7.2,)
```

Comment:
Hi Mario,
I saw you set it as [7.2), so I guess it's not supported on rhel 8.10,right ?
Thanks.

Reply:
@MiriamDeng Indeed, this feature is not supported in RHEL 8.10